docs/70_release.md

```diff
@@ -0,0 +1,114 @@
+# Make a new release
+
+
+Making a new release involves the following step:
+
+  1. Update the CHANGELOG
+  2. make a release branch
+  3. fix the versions in requirements.txt and commit it in the branch
+  4. build DIRACOS
+  5. deploy the archive
+
+
+First of all, get the latest master branch, from which we perform the release. **Careful, you are working with the main repository !**
+
+```
+git clone git@github.com:DIRACGrid/DIRACOS.git
+cd DIRACOS
+```
+
+## Updating the CHANGELOG
+
+This has to be done using [this script](https://github.com/DIRACGrid/DIRAC/blob/integration/docs/Tools/GetReleaseNotes.py) in the DIRAC repository:
+
+Documentation on how to use it can be found [here](https://dirac.readthedocs.io/en/latest/DeveloperGuide/DevelopmentModel/ReleaseProcedure/index.html?highlight=GetReleaseNotes#release-notes)
+
+Once the changelog is updated, commit and push to the master branch.
+
+```
+git add CHANGELOG
```

Comment:
It would be nice if the different packages (DIRAC, DIRACOS, etc.) have the same for the file containing the release notes. I.e. `release.notes` Makes it easier to write one script to treat them all.

Reply:
Fair enough. Do we want to stick with `release.notes` or go for the standard `CHANGELOG` ?